Makefile

```diff
@@ -155,10 +155,16 @@ check-no-changes: ## Check that there are no uncommitted changes
 	@[ "$(GIT_STATUS)" = "" ] || ( echo "There are uncommitted changes:\n$(GIT_STATUS)"; exit 1; )
 
 .PHONY: generate
-generate: $(TFPLUGINDOCS_BIN) $(OAPI_CODEGEN_BIN) $(TERRAFORM_BIN) ## Generate Golang client for the NuoDB REST API and Terraform provider documentation
-	curl -s https://raw.githubusercontent.com/nuodb/nuodb-cp-releases/v$(CP_VERSION)/openapi.yaml -o openapi.yaml
```

Comment:
What is the right way to update `CP_VERSION`?
Previously updating the variable followed by `make generate` would do everything needed. Now it looks like we need to download the openapi spec manually.

Reply:
Good point. I'll move the check of the tag into the `make generate` target so that the spec can be updated by directly invoking `make update-spec`.